.github/jobs/configure-checks/all.bats

```diff
@@ -0,0 +1,349 @@
+   refute_line "    - tmp..............: /opt/domjudge/judgehost/tmp"
+   refute_line "    - judge............: /opt/domjudge/judgehost/judgings"
+   assert_line " * prefix..............: /tmp"
+   assert_line " * documentation.......: /tmp/doc"
+   assert_line " * domserver...........: /tmp/domserver"
+   assert_line "    - libvendor........: /tmp/domserver/lib/vendor"
+   assert_line " * judgehost...........: /tmp/judgehost"
+   assert_line "    - judge............: /tmp/judgehost/judgings"
+}
+
+@test "Check FHS" {
+   setup
+   run run_configure --enable-fhs
+   refute_line " * prefix..............: /opt/domjudge"
+   refute_line " * documentation.......: /opt/domjudge/doc"
+   refute_line " * domserver...........: /opt/domjudge/domserver"
+   refute_line "    - webapp...........: /opt/domjudge/domserver/webapp"
+   refute_line "    - example_problems.: /opt/domjudge/domserver/example_problems"
+   refute_line " * judgehost...........: /opt/domjudge/judgehost"
+   refute_line "    - lib..............: /opt/domjudge/judgehost/lib"
+
+   assert_line " * prefix..............: /usr/local"
+   assert_line " * documentation.......: /usr/local/share/doc/domjudge"
+   assert_line " * domserver...........: "
+   assert_line "    - bin..............: /usr/local/bin"
+   assert_line "    - etc..............: /usr/local/etc/domjudge"
+   assert_line "    - lib..............: /usr/local/lib/domjudge"
+   assert_line "    - libvendor........: /usr/local/lib/domjudge/vendor"
+   assert_line "    - log..............: /usr/local/var/log/domjudge"
+   assert_line "    - run..............: /usr/local/var/run/domjudge"
+   assert_line "    - sql..............: /usr/local/share/domjudge/sql"
+   assert_line "    - tmp..............: /tmp"
+   assert_line "    - webapp...........: /usr/local/share/domjudge/webapp"
+   assert_line "    - example_problems.: /usr/local/share/domjudge/example_problems"
+   assert_line " * judgehost...........: "
+   assert_line "    - bin..............: /usr/local/bin"
+   assert_line "    - etc..............: /usr/local/etc/domjudge"
+   assert_line "    - lib..............: /usr/local/lib/domjudge"
+   assert_line "    - libjudge.........: /usr/local/lib/domjudge/judge"
+   assert_line "    - log..............: /usr/local/var/log/domjudge"
+   assert_line "    - run..............: /usr/local/var/run/domjudge"
+   assert_line "    - tmp..............: /tmp"
+   assert_line "    - judge............: /usr/local/var/lib/domjudge/judgings"
+   assert_line "    - chroot...........: /chroot/domjudge"
+   assert_line "    - cgroup...........: /sys/fs/cgroup"
+}
+
+@test "Alternative dirs together with FHS" {
+   setup
+   run run_configure "--enable-fhs --with-domserver_webappdir=/run/webapp --with-domserver_tmpdir=/tmp/domserver --with-judgehost_tmpdir=/srv/tmp --with-judgehost_judgedir=/srv/judgings --with-judgehost_chrootdir=/srv/chroot/domjudge --with-judgehost_cgroupdir=/sys/fs/altcgroup"
```

Comment:
Nitpick: quoting all the configure options inside a single string works, but only because it is passed to `su -c` in `run_configure` which word expands it again.